src/main/java/hlf/java/rest/client/service/impl/NetworkStatusImpl.java

```diff
@@ -394,11 +423,11 @@ public ResponseEntity<ClientResponseModel> addAnchorPeersToChannel(
             ErrorCode.HYPERLEDGER_FABRIC_TRANSACTION_ERROR,
             ErrorCode.HYPERLEDGER_FABRIC_TRANSACTION_ERROR.name(),
             e);
-      } catch (IOException e) {
-        log.warn("Error while establishing connection to the gateway");
-        throw new ServiceException(
-            ErrorCode.HYPERLEDGER_FABRIC_CONNECTION_ERROR,
-            "Error while establishing connection to the gateway",
+      } catch (Exception e) {
+        log.warn("Error while channel configuration update : " + e.getMessage());
```

Comment:
Use standard log4j formatting instead of concatenating ythe string.
For eg: This should be written as `log.warn("Error while channel configuration update with error message : {}", e.getMessage);`

Reply:
Thanks, I see string concatenation being used at other places as well, I have changed it to log4j formatting in the files where this PR changes are present. We can pick up refactoring to ensure logging format is consistent throughout the data connector, currently it is mixed.